assets/script.js

```diff
@@ -104,7 +104,7 @@ function displayRecipes(url) {
       let recipeDiv = $("<div>")
         .addClass("card-body col-lg-3 col-md-3 col-sm-6 text-center")
         // .css("background-color", "rgb(107,101,75)");
-        .css("background-image", `url(${mealImgURL})`);
+        .css("background-image", `url(${mealImgURL})`)
```

Comment:
Here's where I think it's missing a semi-colon

Reply:
Hi Naomi, yes agree there is a semi-colan missing there, I'll add that